client/src/screens/exercise/view.js

```diff
@@ -193,6 +195,8 @@ export const Exercise = connect({
 }, ctrl => ({
   training: ctrl.getTraining(),
   exercise: ctrl.getExercise(),
+  currentExercise: ctrl.getCurrentExerciseIntoNavbar,
+  lengthExercises: ctrl.getLengthExercisesIntoNavbar,
```

Comment:
1. `lengthExercises` it looks like a property, not a method, change naming (you can simply mirror it as getLengthExercisesIntoNavbar)
2. `getLengthExercisesIntoNavbar` - name contains implementation details, which is bad, `LengthExercises`. also, humans don't say like this too :) what about `getTotalExercises`?

Reply:
the same for currentExercise